docs/api.md

```diff
@@ -158,12 +158,12 @@ as_unschedule_action( $hook, $args, $group )
 ### Parameters
 
 - **$hook** (string)(required) Name of the action hook.
-- **$args** (array) Arguments to pass to callbacks when the hook triggers. Default: _`array()`_.
-- **$group** (array) The group to assign this job to. Default: _''_.
+- **$args** (array) Arguments passed to callbacks when the hook triggers. Default: _`array()`_.
+- **$group** (string) The group the job is assigned to. Default: _''_.
 
 ### Return value
 
-(null)
+(string|null) The scheduled action ID if a scheduled action was found, or null if no matching action found.
```

Comment:
I'm okay with changing the Action ID type to int so long as it's updated to that wherever we make changes.

Reply:
I'm mostly interested in getting the documentation corrected in this PR. I would certainly prefer for this function to return `int` rather than `string`, and also prefer to leave that for another PR.